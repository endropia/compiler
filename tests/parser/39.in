var
    sum: integer;

begin
    sum := 0;

    while sum < 100 do begin
        sum := sum + 3;
    end;

    writeln(sum);
end.