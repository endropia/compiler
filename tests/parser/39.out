program : Unnamed program
   var: 
      sum
      type: integer
   stmts:
      :=
         sum
         0
      while
         <
            sum
            100
            stmts:
               :=
                  sum
                  +
                     sum
                     3
      call
         writeln
            sum
